### ISSUE TITLE
[Feature Request] VTabs fixed-tabs option to align left

### ISSUE THREAD

**Issue by Maxim-Mazurok** [vuetifyjs/vuetify#13451]
### Problem to solve
Material design [spec](https://material.io/components/tabs#fixed-tabs) says:
> Fixed tabs in a cluster can be aligned to the left or right.

But there's no way to disable the center alignment of `<VTabs fixed-tabs>` without using external CSS.

### Proposed solution
Either add prop, like `align="right|left|center"` with `center` being default for backward compatibility.

Or, expose these margins as variables:
```
.v-tabs--fixed-tabs > .v-tabs-bar,
.v-tabs--centered > .v-tabs-bar
  .v-tabs-bar__content > *:last-child
    +ltr()
      margin-right: auto

    +rtl()
      margin-left: auto

  .v-tabs-bar__content > *:first-child:not(.v-tabs-slider-wrapper),
  .v-tabs-slider-wrapper + *
    +ltr()
      margin-left: auto

    +rtl()
      margin-right: auto
```
that we could override. Similar to $tabs-item-align-with-title-margin, etc.

**Comment by Maxim-Mazurok** [comment]
@johnleider this issue doesn't seem to be addressed in v3.3.19: https://play.vuetifyjs.com/#eNrFUcsKwjAQ/JWwZ9si9aJUwa/wYD2k6fYBaVKSbVXEfzdNKyj4unnanZ1h2GH2F7BGRNu2DfsOYQUJYdNKTrhJFWNJHwhucr97RDyzrKhPmI9rVgZCS23WKbSmbrg5p8C4rEvleXe2xA2lMFncTVjPZYeO1goduUNmK93JPIk8/U5NR+3UGTKtGFX4TV0ZHNwlFvQsnZD9KZhFoVX+IppARWj+m2384UO6AUwdJtFDuXCdQRzG4QLGOV/C4QYFCafm

However, I see that in 3.4.0-alpha.1 there's no space around the tabs, so alignment becomes irrelevant, is that why the issue was closed?
https://play.vuetifyjs.com/#eNrFUcsKwjAQ/JWwZ5sqepIq+BUerIdtutpCTEKyrYr47/YlKPi6edqdnWHYYTYXCF7FK+dkXRHMIWE6OI1My9QIkdSRQp93e4cYsyB25Ynyfs32kbLa+kUKzpcH9OcUBOpybzq+OQdGzykMFncTUaOuqKGtoYZckwiFrXSexB39Ts1H26gzEtYILuibuvDUumva8bN0QOGnYIGUNfmLaIoMk/9vtv6HD+laMHSYxA/lwnUEUzmVM2jnTI4j1K5AOYHtDagZqj8=

Thanks!

**Comment by johnleider** [comment]
Please don't comment on old closed issues.
